care/facility/tests/test_patient_api.py

```diff
@@ -82,6 +84,29 @@ def _get_disease_state_representation(self, disease_state):
             return DiseaseStatusEnum(disease_state).name
         return disease_state
 
+    def _get_metainfo_representation(self, patient: PatientRegistration):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5344113)